concent_api/core/message_handlers.py

```diff
@@ -571,6 +550,28 @@ def handle_send_force_subtask_results(
             subtask_id=task_to_compute.compute_task_def['subtask_id'],
         )
 
+        if subtask is not None and subtask.state_enum == Subtask.SubtaskState.FORCING_ACCEPTANCE:
+            return message.concents.ServiceRefused(
+                reason=message.concents.ServiceRefused.REASON.DuplicateRequest,
+            )
+
+        (claim_against_requestor, claim_against_provider) = bankster.claim_deposit(
+            subtask_id=task_to_compute.subtask_id,
+            concent_use_case=ConcentUseCase.FORCED_ACCEPTANCE,
+            requestor_ethereum_address=task_to_compute.requestor_ethereum_address,
+            provider_ethereum_address=task_to_compute.provider_ethereum_address,
+            subtask_cost=task_to_compute.price,
+            requestor_public_key=requestor_public_key,
+            provider_public_key=provider_public_key,
+        )
+
+        assert claim_against_provider is None
+
+        if claim_against_requestor is None:
+            return message.concents.ServiceRefused(
+                reason=message.concents.ServiceRefused.REASON.TooSmallRequestorDeposit,
+            )
```

Comment:
I think Bankster should return a flag from `claim_deposit()` to signal this case. Otherwise there's no way for that function to simply say there's nothing to pay.

This was not in the original blueprint, but we should change it. We can do it in a separate pull request though.

Reply:
But if `None` means "the deposit is too small, refuse service" then what is the value for "there's nothing to pay, proceed"? It can't mean both.

Reply:
This situation should not appear. Subtask cost should be always greater than 0 so there will be always something to pay for.

Reply:
Yes, but we agreed it will be done in separate issue.